htdocs/langs/en_US/banks.lang

```diff
@@ -148,7 +148,7 @@ BackToAccount=Back to account
 ShowAllAccounts=Show for all accounts
 FutureTransaction=Transaction in futur. No way to conciliate.
 SelectChequeTransactionAndGenerate=Select/filter checks to include into the check deposit receipt and click on "Create".
-InputReceiptNumber=Choose the bank statement related with the conciliation. Use a sortable numeric value (such as, YYYYMM)
+InputReceiptNumber=Choose the bank statement related with the conciliation. Use a sortable numeric value: YYYYMM or YYYYMMDD
```

Comment:
My concern is to let people know that if they use a basic numbering scheme like "1, 2, 3, 4...." then the bank statement won't be ordered correctly. Right now the InputReceiptNumber message is confusing and it leads people to believe that  "1, 2, 3, 4...." is a "sortable numeric value".


InputReceiptNumber=Choose the bank statement related with the conciliation. If you want the statements to be ordered correctly, please use one of the following format: YYYYMM or YYYYMMDD